index.html

```diff
@@ -8994,7 +8994,7 @@ <h3>Processing actions</h3>
   of <a>getting the property</a> <code>y</code>
   from <var>action item</var>.
 
- <li><p>If <var>y</var> is not an <a>Integer</a>, return <a>error</a>
+ <li><p>If <var>y</var> is not an <a>Number</a>, return <a>error</a>
```

Comment:
Not sure if we should also change that for wheel input sources. The `scroll` action has x and y positions as well. But for now in Firefox we only allow fractional values for pointer actions. @OrKoN how does it look like for Chrome?

Reply:
I believe Chrome supports non-integer input for the scroll/wheel event

Reply:
We have to follow-up internally first. That means lets split it up and get the changes for pointers landed first.